fa-products-service-sand-ne-001/http-get-product-by-id/function.json

```diff
@@ -0,0 +1,21 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": [
+        "get",
+        "post"
```

Comment:
It's redundant as this functions only reruns a product by id and doesn't save any